lib/shared/cliOptions.js

```diff
@@ -47,12 +47,6 @@ module.exports = {
     desc: chalk.gray(
       'Print the task dependency tree for the loaded gulpfile.'),
   },
-  depth: {
```

Comment:
I rethought about flag names that `--depth`, `--compact` and `--sort` are better than `--tasks-depth`, `--compact-tasks` and `--sort-tasks`. When we use these flags, we type a command like `gulp --tasks --tasks-depth`. The word "tasks" is redundant for users, I think.


I think good about the config names, but it is possible to group these flags like `flags.tasks.depth`, `flags.tasks.sort`, `flags.tasks.compact` (in .gulp.json, `{ flags: { tasks: { depth: 3, sort: true, compact: true } } }`).

What do you think?